### PR TITLE
Login func

### PR DESCRIPTION
**Pull request by MarvinMichel**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ require('dotenv').config();
 const express = require('express');                                               // Marvin
 const app = express();                                                            // Marvin
 const port = 8000;                                                                // Marvin
-const bodyParser = require('body-parser');                                        // Marvin
-const urlencodedParser = bodyParser.urlencoded({ extended: true });               // Marvin
```
Comment (JadeSsst) [inline server.js:6]: This has to stay, right? Because i need it but it will be pushed in when my pull request merges so no worries

Comment (MarvinMichel) [reply server.js:6]: I'm thinking of replacing body-parser. I found out that express has a body-parser too. I just need a little research to find out how we could implement it trough out the server.

Comment (JadeSsst) [reply server.js:6]: Okay that makes sense!

File: server.js [hunk 2]
```diff
@@ -26,22 +28,23 @@ mongoose.connect(
...
-        firstName: firstName,
-        lastName: lastName
+        firstName,
+        lastName
       },
-      age: age,
-      gender: gender
+      age,
+      gender,
+      picture
```
[the remark this answers is not in the document]
Comment (MarvinMichel) [reply server.js:47]: It's right, the user model will be changed in the final version. This was just a test version, no worries!

**Pull request merged**